### PR TITLE
Upgrade AWS provider to 3.34.0

### PR DESCRIPTION
**Pull request by gedejong**
Upgrades to AWS Provider 3.34.0.

### CODE REVIEW

**Comment by cloudmaniac** [comment]
Thx @gedejong! May I ask a few questions before merging your commits?

A first one here, and the remaining ones in the code: why the `3.34.0` version of the aws provider? Is that because it's the latest or for something else?

**Comment by gedejong** [comment]
> Thx @gedejong! May I ask a few questions before merging your commits?
> 
> A first one here, and the remaining ones in the code: why the `3.34.0` version of the aws provider? Is that because it's the latest or for something else?

Yes, the ~> 3.34.0 indicates that the module has been tested with 3.34.0 and it is the latest.

File: main.tf [hunk 1]
```diff
@@ -18,8 +19,6 @@ data "aws_route53_zone" "main" {
 ## ACM (AWS Certificate Manager)
 # Creates the wildcard certificate *.<yourdomain.com>
 resource "aws_acm_certificate" "wildcard_website" {
-  provider = aws.us-east-1 # Wilcard certificate used by CloudFront requires this specific region (https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-requirements.html)
```
Comment (cloudmaniac) [inline main.tf:21]: Why removing the preferences to the `us-east-1` provider? This region is required for ACM if used by Cloudfront.

Comment (cloudmaniac) [inline main.tf:21]: Why removing the preferences to the `us-east-1` provider @edwinbdr ? This region is required for ACM if used by Cloudfront.

**Review by cloudmaniac — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (cloudmaniac) [reply main.tf:21]: I confirm it doesn't work w/o that region. 

Screenshot 2021-03-29 at 22 39 46

**Pull request merged**